src/base/NekRSProblem.C

```diff
@@ -488,6 +568,11 @@ NekRSProblem::checkInitialFluxValues(const Real & nek_flux, const Real & moose_f
 void
 NekRSProblem::sendVolumeDeformationToNek()
 {
+  double disp_scale = 1.0;
+
+  if (_nondimensional)
```

Comment:
If `nondimensional = false`, the `nekrs::solution::referenceLength()` is set to a default value of 1, so we can use simpler logic here and just divide lines 619-621 straight away by `nekrs::solution::referenceLength()`.

Reply:
It's also available as `_L_ref`